selftests/functional/utils/test_podman.py

```diff
@@ -16,3 +16,22 @@ def test_python_version(self):
         coro = podman.get_python_version('fedora:34')
         result = loop.run_until_complete(coro)
         self.assertEqual(result, (3, 9, '/usr/bin/python3'))
+
+    def test_container_info(self):
+        """
+        :avocado: dependency={"type": "package", "name": "podman", "action": "check"}
+        :avocado: tags=slow
+        """
+        podman = Podman()
+        loop = asyncio.get_event_loop()
+        coro = podman.execute("create", "fedora:latest", "/bin/bash")
```

Comment:
The previously existing test uses "fedora:34".  To avoid downloading multiple images, I would recommend using the same.